charts/newrelic-k8s-metrics-adapter/templates/_helpers.tpl

```diff
@@ -1,110 +1,22 @@
-{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-{{- end -}}
-{{- end -}}
-
-{{/*
-Create chart name and version as used by the chart label.
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
-{{/*
-Common app label
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.appLabel" -}}
-app.kubernetes.io/name: {{ include "newrelic-k8s-metrics-adapter.name" . }}
-{{- end -}}
 
-{{/*
-Common labels
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.labels" -}}
-{{ include "newrelic-k8s-metrics-adapter.appLabel" . }}
-helm.sh/chart: {{ include "newrelic-k8s-metrics-adapter.chart" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end -}}
-
-{{/*
-Create the name of the service account to use
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create -}}
-    {{ default (include "newrelic-k8s-metrics-adapter.fullname" .) .Values.serviceAccount.name }}
-{{- else -}}
-    {{ default "default" .Values.serviceAccount.name }}
-{{- end -}}
-{{- end -}}
-
-{{/*
-Return the licenseKey
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.licenseKey" -}}
-{{- if .Values.global}}
-  {{- if .Values.global.licenseKey }}
-      {{- .Values.global.licenseKey -}}
-  {{- else -}}
-      {{- .Values.licenseKey | default "" -}}
-  {{- end -}}
-{{- else -}}
-    {{- .Values.licenseKey | default "" -}}
-{{- end -}}
-{{- end -}}
-
-{{/*
-Return the cluster
-*/}}
-{{- define "newrelic-k8s-metrics-adapter.cluster" -}}
-{{- if .Values.global -}}
-  {{- if .Values.global.cluster -}}
-      {{- .Values.global.cluster -}}
-  {{- else -}}
-      {{- .Values.cluster | default "" -}}
-  {{- end -}}
-{{- else -}}
-  {{- .Values.cluster | default "" -}}
-{{- end -}}
+{{- /* Allow to change pod defaults dynamically based if we are running in privileged mode or not */ -}}
+{{- define "common.securityContext.podDefaults" -}}
+fsGroup: 1001
+runAsUser: 1001
+runAsGroup: 1001
 {{- end -}}
 
 {{/*
 Select a value for the region
+When this value is empty the New Relic client region will be the default 'US'
 */}}
 {{- define "newrelic-k8s-metrics-adapter.region" -}}
 {{- if .Values.config.region -}}
   {{- .Values.config.region | upper -}}
-{{- else if .Values.global -}}
-  {{- if .Values.global.nrStaging -}}
-    Staging
-  {{- else if eq (include "newrelic-k8s-metrics-adapter.licenseKey" . | substr 0 2) "eu" -}}
-    EU
-    {{- end }}
-{{- else if eq (include "newrelic-k8s-metrics-adapter.licenseKey" . | substr 0 2) "eu" -}}
+{{- else if (include "common.nrStaging" .)  -}}
+Staging
+{{- else if eq (include "common.license._licenseKey" . | substr 0 2) "eu" -}}
```

Comment:
We could use `contains` or `hasPrefix` to check this http://masterminds.github.io/sprig/strings.html

Reply:
I used hasPrefix, thanks for the link, I'll added to my toolset